thrust/system/cuda/detail/reduce_by_key.h

```diff
@@ -1079,12 +1079,13 @@ namespace __reduce_by_key {
 
     size_type num_items = thrust::distance(keys_first, keys_last);
 
+    pair<KeysOutputIt, ValuesOutputIt> result = thrust::make_pair(keys_output, values_output);
+
```

Comment:
Tiniest of nitpicks, that line is a bit much.

No change requested

Reply:
I've split it but our new line limit (100) changed the line to this state :)